tasks/binder-at-home.tex

```diff
@@ -34,12 +34,14 @@
 The compute resources offered through mybinder are modest: at most 2 GB of RAM
 and a single CPU core (\TODO{check this!}).
 Most laptops and Desktops have similar or much better hardware capabilities than
-the mybinder cloud-computing resources currently offer
+the mybinder cloud-computing resources currently offer.
+
+For some research area, it is essential to access Big data sets to reproduce the scientific computing workflow.  For such cases, it is essential to reproduce the work, using infrastructure have optimised access to the data, so that one does not consume unessesary computing and network resources.  
```

Comment:
```suggestion
For some research areas, it is essential to access big data sets to reproduce the scientific computing workflow.  For such cases, it is essential to reproduce the work using infrastructure that has optimised access to the data, so that one does not consume unnessesary computing and network resources.  
```